src/viam/services/motion/client.py

```diff
@@ -162,6 +218,106 @@ async def move_on_map(
+
+        request = StopPlanRequest(
+            name=self.name,
+            component_name=component_name,
+            extra=dict_to_struct(extra),
+        )
+        _: StopPlanResponse = await self.client.StopPlan(request, timeout=timeout)
+        return
+
+    async def get_plan(
+        self,
+        component_name: ResourceName,
+        last_plan_only: Optional[bool] = None,
+        execution_id: Optional[str] = None,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> GetPlanResponse:
+        """**Experimental**
+        Returns the plan(s) & state history of the most recent execution to move a component.
+
+        Returns a result if the last execution is still executing OR changed state within the
+        last 24 hours AND the robot has not reinitialized.
+        Plans are never mutated.
+        Replans always create new plans.
+        Replans share the execution_id of the previously executing plan.
+
+        Args:
+            component_name (ResourceName): The component to stop
+            last_plan_only (Optional[bool]): If supplied, the response will only return the last plan for the component / execution
+            execution_id (Optional[str]): If supplied, the response will only return plans with the provided execution_id
+
+        Returns:
+            ``GetPlanResponse`` (GetPlanResponse): The current PlanWithStatus & replan history which matches the request
+        """
+        if extra is None:
+            extra = {}
+
+        if last_plan_only is None:
+            last_plan_only = False
+        request = GetPlanRequest(
+            name=self.name,
+            component_name=component_name,
+            last_plan_only=last_plan_only,
+            execution_id=execution_id,
+            extra=dict_to_struct(extra),
+        )
+        response: GetPlanResponse = await self.client.GetPlan(request, timeout=timeout)
+        return response
+
+    async def list_plan_statuses(
+        self,
+        only_active_plans: Optional[bool] = None,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> ListPlanStatusesResponse:
+        """**Experimental**
+        Returns the status of plans created by MoveOnGlobe requests that are executing
+        OR are part of an execution which changed it statewithin the a 24HR TTL OR until the robot reinitializes.
+
+        Args:
+            only_active_plans (Optional[bool]):  If supplied, the response will filter out any plans that are not executing
+
+        Returns:
+            ``ListPlanStatusesResponse`` (ListPlanStatusesResponse): List of last known statuses with the
+            associated IDs of all plans within the TTL ordered by timestamp in ascending order
+        """
+        if extra is None:
+            extra = {}
+
+        if only_active_plans is None:
+            only_active_plans = False
```

Comment:
Why not default it to `False` instead of `None`?

Reply:
Fixed